src/tools/nwc/get_balance.ts

```diff
@@ -10,20 +10,25 @@ export function registerGetBalanceTool(
     "get_balance",
     {
       title: "Get Balance",
-      description: "Get the balance of the connected lightning wallet. Amounts are in millisats (1000 millisats = 1 sat). Preferred human-readable unit is sats.",
+      description: "Get the balance of the connected lightning wallet",
       outputSchema: {
-        balance: z.number().describe("Current wallet balance in millisats"),
+        balance: z.number().describe("Current wallet balance in sats"),
       },
     },
     async () => {
       const balance = await client.getBalance();
 
+      // Convert millisats to sats
+      const convertedBalance = {
+        balance: Math.ceil(balance.balance / 1000), // Round up when converting millisats to sats
```

Comment:
would it help some robot if the name of the property is more descriptive  `balance_in_sats` ?